challenges/dog-photo-gallery/script.js

```diff
@@ -0,0 +1,61 @@
+const gallery = document.createElement("div");
+gallery.classList.add("photo-gallery");
+const displayDogBtn = document.createElement("button");
+displayDogBtn.textContent = "Click to display random dog";
+const ul = document.createElement("ul");
+const loadMsgEl = document.createElement("div");
+loadMsgEl.classList.add("loading-msg");
+
+document.body.append(gallery, displayDogBtn, ul, loadMsgEl);
+
+class DogGallery {
```

Comment:
how would this code look like if the DogGallery class was in another file? if you have many classes, is it easier to read code with everything in one file, or classes being in separate files?

Reply:
separate files would be much better.